internal/pkg/services/graphql-service_test.go

```diff
@@ -0,0 +1,49 @@
+package services
```

Comment:
I noticed that the GraphQL file is using `-` in its filename. By convention, we should use only `_`.

Reply:
I'll shorten the names to avoid having to use a separator at all and leave the `build` and `workflows` directories as is